python/housinginsights/sources/wmata_distcalc.py

```diff
@@ -0,0 +1,234 @@
+import sys
+import csv
+import json
+import requests
+import os
+import time
+#import psycopg2
+import time
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),os.pardir)))
+from housinginsights.sources.base import BaseApiConn
+import housinginsights.tools.dbtools as dbtools
+import sqlalchemy
+#sys.path.insert(0, '../tools/dbtools.py')
+#import tools.dbtools
+
+class WmataApiConn(BaseApiConn):
+
+    def __init__(self):
+
+        secretsFileName = "../secrets.json"
+
+        now = time.strftime("%Y%m%d")
+        outputDir = "../../../data/raw/wmata/" + now
+
+        if not os.path.exists(outputDir):
+            os.makedirs(outputDir)
+
+        self.distOutputFileName = outputDir + "/dist.csv"
+        self.infoOutputFileName = outputDir + "/wmatainfo.csv"
+
+        print("Will write WMATA_DIST table to {}".format(self.distOutputFileName))
+        print("Will write WMATA_INFO table to {}".format(self.infoOutputFileName))
+
+        #pull API keys
+        self.api_keys = json.loads(open(secretsFileName).read())
+        self.wmata_api_key = self.api_keys['wmata']['api_key']
+        self.mapbox_api_key = {'access_token':self.api_keys['mapbox']['public-token']}
+
+        #write out the wmata info csv
```

Comment:
Should probably move these csv initialization lines into get_data. Currently it's not a concern because we instantiate the object and then use it immediately, but there might be situations where we want to call get_data later or in a different context.